src/inject-babel-helpers.ts

```diff
@@ -0,0 +1,64 @@
+import * as dom5 from 'dom5';
+import {fs} from 'mz';
+import * as parse5 from 'parse5';
+import * as path from 'path';
+
+import {AsyncTransformStream, getFileContents} from './streams';
+
+import File = require('vinyl');
+
+const p = dom5.predicates;
+
+const scriptOrImport = p.OR(
+    p.hasTagName('script'),
+    p.AND(p.hasTagName('link'), p.hasSpaceSeparatedAttrValue('rel', 'import')));
```

Comment:
I added personal TODO to update other html import selectors to use hasSpaceSeparatedAttrValue.  Thanks for doing that here.